File: Basic/problem_9.c
/*
Write a program to Input two numbers and swap them.
*/
// Solved by Ashwani2529
x =int(input("Enter first number: "))
y =int(input("Enter second number: "))
temp = x
x = y
y = temp
print('First number after swapping: {}'.format(x))
print('Second number after swapping: {}'.format(y))

File: Basic/Python/problem_9.py
# 9. Input two numbers and swap them.
# by Nitin Kumar


def swap(n1, n2):
    print('Numbers before swapping:\n n1 = {} & n1 = {}'.format(n1,n2))
    n1,n2 = n2,n1
    print('Numbers after swapping:\n n1 = {} & n1 = {}'.format(n1,n2))

n1 = int(input('Enter 1st number:'))
n2 = int(input('Enter 2nd number:'))
swap(n1, n2)
